Build Json fetching

chen678 opened the pull request:
The server will return hard-coded ArrayList as fake comments data, and `/js/script.js` should be fetching this data and using it to add HTML elements to the page. The console will print this date as well to show its reception.

Web page screenshot:
![Screen Shot 2020-02-27 at 7 04 57 PM](https://user-images.githubusercontent.com/43182567/75498312-cc8c4c80-5994-11ea-8a5d-379b52585920.png)
![Screen Shot 2020-02-27 at 7 05 04 PM](https://user-images.githubusercontent.com/43182567/75498314-cd24e300-5994-11ea-9944-70854e5c3e73.png)

kmontgomery commented on portfolio/src/main/java/com/google/sps/servlets/DataServlet.java:
```diff
@@ -14,19 +14,40 @@
+    response.getWriter().println(json);
+  }
+
+  private ArrayList<String> createArrayList(){
```
Can you add a more descriptive name to this method?

kmontgomery commented on portfolio/src/main/webapp/js/script.js:
```diff
@@ -12,17 +12,22 @@
+    }
 }
+
+function myFetch(){
```
Add javadoc describing what this method does to help the reader understand.

kmontgomery commented on portfolio/src/main/webapp/js/script.js:
```diff
@@ -12,17 +12,22 @@
-  // Add it to the page.
-  const greetingContainer = document.getElementById('greeting-container');
-  greetingContainer.innerText = greeting;
+// Todo: implement a working responsive top navigation tabs
```
Nice job leaving a TODO!

kmontgomery commented on portfolio/src/main/webapp/index.html:
```diff
@@ -2,17 +2,86 @@
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
+  
+  <body onload = "myFetch()">
```
Rename this method so that the reader can have a better understanding of what it does.

chen678 commented:
addressed comments. 
![Screen Shot 2020-02-27 at 9 06 05 PM](https://user-images.githubusercontent.com/43182567/75504009-3c0a3800-59a5-11ea-9d84-6786721d615d.png)

chen678 commented:
@kmontgomery thanks for reviewing @night!
